packages/venia-ui/lib/components/HomePage/homePage.css

```diff
@@ -80,6 +80,7 @@
  * Target the slider.
  */
 :global(.venia-home-slider .slick-slider) {
+    min-height: 628px !important;
```

Comment:
How are these min-height decided, and can we work with device height instead?

Reply:
This is the height of the content with the height of the pagination.
Since I couldn't update the min-height from the backend, CSS was the only way to apply it for the mobile and desktop versions and avoid a div without any height.